code/src/components/GlobalStyles.js

```diff
@@ -0,0 +1,39 @@
+import styled, { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+
+`
+
+export const OuterWrapper = styled.div`
+  background-color: #faf6f0;
```

Comment:
Add this to the html or body. This will remove the edge if the content expands further down the page :)